cloudsync/sync/state.py

```diff
@@ -369,6 +376,8 @@ def paths_differ(self, side):
 
     def needs_sync(self):
         for i in (LOCAL, REMOTE):
+            if self[i].force_sync:
+                return True
```

Comment:
Why reimplement `needs_sync` in the `SyncEntry` class when it is already implemented in the `SideState` class, and implemented differently? Why not just
 ``` 
 return self[0].needs_sync() or self[1].needs_sync()
```
as the whole `SyncEntry.needs_state` method?

Reply:
left it like this originally because the logic is slightly different between the 2 needs_sync() impls, but your suggestion makes sense (code reuse is a good thing) and tests still pass, so I will make the change.